service/src/main/resources/application.yml

```diff
@@ -37,3 +37,4 @@ listener:
     allowHeaders: "Authorization, Content-Type, Accept, Origin,X-App-Id"
     maxAge: "1728000"
     contentSecurityPolicy: "frame-ancestors http://localhost:3000;report-uri https://terra.report-uri.com/r/d/csp/reportOnly"
+    validHosts:
```

Comment:
what does the empty inclusion here do, is this populated elsewhere?

Reply:
Not entirely sure - I had seen it elsewhere (Can't remember off the top of my head) but I figured it's a reminder placeholder that we expect a value there.

Do you have a suggestion for a default value?

Reply:
Is this where we should define the valid hosts allowlist?

Reply:
We COULD have some default values here, but it will be overwritten by arguments passed to the listener's docker command.

Personally, I think it's at least worthwhile to keep here as a reminder that it needs values.